students/Ben_freeman/lesson05/mailroom3.py

```diff
@@ -0,0 +1,84 @@
+import sys
+#Donor dict
+donor_list={
+"Ahmed":[123,456,789],
+"Sally": [10,11],
+"Bob": [12],
+"Vladamir": [13.14,1516.17],
+"Santa": [181920.21,22,23.24],
+}
+
+
+#send thanks funct
+def send_thanks():
+    while True:
+        query=input("Please enter a Full Name, enter 'list' to see a list of current donors, or quit to exit: ")
+
+        if query=="list":
+            print('\n'.join(donor_list))
+        elif query.lower()=="quit":
+            return
+        else:
+            tracker=0
+            while tracker==0:
+                try:
+                    new_donate=float(input("\nHow much would you like to donate: \n"))
+                    tracker=1
```

Comment:
you can avoid having a "flag" variable (tracker)

just a add a `break` on line 26 instead of `tracker=1`